core/chains/evm/client/errors.go

```diff
@@ -160,6 +160,19 @@ var harmony = ClientErrors{
 	Fatal:                   harmonyFatal,
 }
 
+// Klaytn
+// https://github.com/klaytn/klaytn/blob/dev/blockchain/error.go
+// https://github.com/klaytn/klaytn/blob/dev/blockchain/tx_pool.go
+var klaytn = ClientErrors{
+	NonceTooLow:                 regexp.MustCompile(`(: |^)nonce too low$`),
+	TransactionAlreadyInMempool: regexp.MustCompile(`(: |^)(known transaction|there is another tx which has the same nonce in the tx pool)`),
+	TerminallyUnderpriced:       regexp.MustCompile(`(: |^)(transaction underpriced|^intrinsic gas too low)`),
+	LimitReached:                regexp.MustCompile(`(: |^)txpool is full`),
+	InsufficientEth:             regexp.MustCompile(`(: |^)insufficient funds`),
```

Comment:
```suggestion
	InsufficientEth:             regexp.MustCompile(`(: |^)insufficient funds|insufficient funds of the sender for value `),
```
During our tests we noticed `insufficient funds of the sender for value ` message as well.

Reply:
I didn't add `$` at the end of `insufficient funds`, so it can cover your case also.